README.md

```diff
@@ -37,15 +37,18 @@ go build .
 
 ## Test the application in new terminal window
 
+Note: For repeatability of the following example, document IDs are hardcoded.
+To generate random ID, just remove "_id" field from the request.
+
 ### Insert users
 
 Run following commands to create two users: Jane and John
 
 ```shell
 curl -X POST localhost:8080/users/create -H 'Content-Type: application/json' \
```

Comment:
for reference: for Java's example I am going to update the requests to not include ID in it.

Reply:
Yeah, I'll do the same in the docs.
Here I just wanted example to be repeatable.
Just copy paste the commands and execute.

Reply:
But I don't like how example looks.